### ISSUE TITLE
Does the pretrained model train on ArcFace?

### ISSUE THREAD

**Issue by Po-Shin** [TreB1eN/InsightFace_Pytorch#61]
Since the name of the file is **IR-SE50 @ BaiduNetdisk**, and  **IR-SE50 @ Onedrive**, may I ask that this is trained on ms1m-refine-v2 with ArcFace? Or just ms1m-refine-v2 with Cross-entropy loss?

Thanks!
